dart/gui/osg/Viewer.hpp

```diff
@@ -114,10 +115,9 @@ class ViewerAttachment : public virtual ::osg::Group
 class Viewer : public osgViewer::Viewer, public dart::common::Subject
 {
 public:
-
   /// Constructor for dart::gui::osg::Viewer. This will automatically create the
   /// default event handler.
-  Viewer(const ::osg::Vec4& clearColor = ::osg::Vec4(0.9,0.9,0.9,1.0));
+  Viewer(const ::osg::Vec4& clearColor = ::osg::Vec4(0.9,0.9,0.9,1.0), bool shadowsOn = false, ::osg::ref_ptr<osgShadow::ShadowTechnique> shadowTechnique = nullptr);
```

Comment:
Would it make sense to get rid of the `shadowsOn` flag and allow a `nullptr` for `shadowTechnique` to indicate that shadows should be off? Then if someone passes in a non-null value for `shadowTechnique`, we have it default to shadows being on?

Also, if there's a default native `osgShadow::ShadowTechnique` that we can recommend to users, it would be nice if we referred to it in the documentation of this function.